apps/web/playwright/api/auth/security.spec.ts

```
@@ -98,35 +99,42 @@ test.describe("Authentication Security Tests - Vulnerability Prevention", () =>
+      expect(responseTime).toBeLessThan(5000);
     });
   });
 
   test.describe("Timing Attack Prevention - User Enumeration Protection", () => {
     test("should not reveal user existence through response timing differences", async ({ request }) => {
+      // Sub-100ms enumeration-timing differences can't be measured meaningfully through Playwright
+      // service mode's remote-browser tunnel — tunnel network jitter dwarfs the signal and makes the
+      // assertion flaky. Keep it running in local mode, where the measurement is valid.
+      // (PLAYWRIGHT_SERVICE_URL is the Azure Playwright service endpoint — present only in service mode.)
+      test.skip(
+        Boolean(process.env.PLAYWRIGHT_SERVICE_URL),
```

ESLint

[error] PLAYWRIGHT_SERVICE_URL is not listed as a dependency in turbo.json

(turbo/no-undeclared-env-vars)